### ISSUE TITLE
overwrite previous results?

### ISSUE THREAD

**Issue by jeffchen2000** [cbg-ethz/shorah#57]
when I run shorah.py, for some reason the program crashed or I terminated the program, and then I re-run shorah.py (with some parameters changed), will the previous generated intermediate files being overwritten? (or being re-used?)

**Comment by DrYak** [comment]
Yes, shorah will only run the dirichlet sampling and regenerate the corrected reads for windows wich were'nt completed during the previous run. 

You should delete the `corrected/` directory to force re-running all the dirichlet sampling.

For a completely clean directory (save for the log files):

    rm -rf corrected coverage.txt debug  freq raw_reads sampling support
